src/python/qeqiskit/backend/backend.py

```diff
@@ -153,6 +159,7 @@ def run_circuitset_and_measure(self, circuitset, **kwargs):
         for i, ibmq_circuit in enumerate(ibmq_circuitset):
             job = jobs[int(i / self.batch_size)]
             circuit_counts = job.result().get_counts(ibmq_circuit)
+            self.number_of_jobs_run += 1
```

Comment:
Just noting that these are good to keep since the batching system makes things different than the default behavior!